src/Features/CSharpTest/ReplacePropertyWithMethods/ReplacePropertyWithMethodsTests.cs

```diff
@@ -1738,13 +1748,6 @@ internal interface ILanguageServiceHost
                 ///     An value that provides access to the language service for the active configured project.
                 /// </returns>
                 object GetActiveProjectContext();
-
-                /// <summary>
```

Comment:
> /// <summary> [](http://example.com/codeflow?start=15&length=14)

I can see an argument where the duplicated comments here provided value.

Reply:
yeah, i thought about it, but decided it was something the user would need to take care of to decide how they wanted the existing comment to map over to multiple members.